content/tutorials/encryption-realtime.textile

```diff
@@ -0,0 +1,618 @@
+  # Notice, how easily we enabled the decryption mechanism simply by using a @channelOpts@ containing our cipher key.
+
+  "See this step in Github":https://github.com/ably/tutorials/commit/encryption-js-step4
+
+blang[nodejs].
+
+  Now that we have seen how to publish an encrypted message to a channel, let's see how we can retrieve it.
+  For this purpose, let's create a @subscriber.js@, with following :
+
+  ```[nodejs]
+    const Ably = require('ably');
+    const constants = require('./constants');
+
+    /* Instance the Ably library */
+    const realtime = new Ably.Realtime({key: constants.API_KEY});
+    const channelOpts = {cipher: {key: constants.SECRET}};
+    const subChannel = realtime.channels.get('encrypted:messages', channelOpts);
+
+    console.log('Retrieving encrypted message');
+
+    /* Subscribe to messages */
+    subChannel.subscribe(console.log);
+  ```
+
+  **What is going on in the code block above?**
+
+  We obtain a @subChannel@ instance very similar to how we did for @pubChannel@.
+  The only difference being, in this instance we'll subscribe to the channel instead of publishing to it, using the @Channel.subscribe()@
+  For simplicity we're passing a @console.log@ to the subscribe() to log each received message.
+  In a real world project you would do more than just log the incoming message, and can define a more appropriate callback per your requirements.
+  But notice how easily we enabled the decryption mechanism, merely by specifying a @channelOpts@ containing our cipher key, during our channel instantiation.
+
+  Let's add one more npm script to our @package.json@ to be able to run our subscriber code from the terminal:
+
+  ```[json]
+    "scripts": {
+        "app:publish": "node ./publisher.js",
+        "app:subscribe": "node ./subscriber.js"
+    },
+  ```
+
+  "See this step in Github":https://github.com/ably/tutorials/commit/encryption-nodejs-step4
+
+We're done, it's that simple. We have now shown you how to install Ably, then we published an encrypted message on a channel, and later retrieved it using the encryption API.
+
+h2(#live-demo). Live demo
+
+In this demo we use the "encryption API":/realtime/encryption to publish and retrieve encrypted messages.
+
+<div id="qr-code" style="float: right; margin: 0 0 15px 15px; width: 200px; display: none; text-align: center">
+  <div style="margin-top: 5px">Scan this on your mobile</div>
+</div>
+
+<a href="#" id="new-browser" target="_blank">Open this demo in a new browser window</a> to see publishing & subscribing of encrypted messages in action.  Alternatively, scan the QR code to see the published message decrypted and delivered in real time.
+
+<div>
+  <div>
+    Message: <input type="text" id="message-text" value="James, call me ASAP - M"> <button id="send-message">Send a message</button>
+  </div>
```

Comment:
In the live demo, it's not clear currently if the message displayed is the published one or received from the subscription. Please improve the log to clarify the same.